steps/add_a_list_card_steps.py

```diff
@@ -158,3 +161,77 @@ def validate_label_name_and_color(context):
     context.list_card = AddAListCardPage(context.driver)
     for i in range(len(total_label_names)):
         context.list_card.verify_color_name_of_label(total_colors[i], total_label_names[i])
+
+
+@when(u'Create lists "{input}" "{filename}" and validate lists')
+def create_list_title_validate_list_name(context, input, filename):
+    context.list_card = AddAListCardPage(context.driver)
+    file_path = os.path.join(os.getcwd(), "resources")
+    context.filename = filename
+    context.listfilepath = os.path.join(file_path, filename)
+    value = int(input)
+    with open(context.listfilepath, 'w') as file:
+        file.write('')
+    for i in range(value):
+        name = fake.name()
+        with open(context.listfilepath, 'a') as file:
+            file.write(f'{name}\n')
+    with open(context.listfilepath, 'r') as file:
+        lines = file.readlines()
+        for line in lines:
+            name = line.strip()
+            context.list_card.enter_list_title(name)
+            context.list_card.click_on_add_a_list_button()
+            context.list_card.verify_added_list_name(name)
+
+
+@when(u'Create cards "{list_size}" "{card_size}" "{filename}" and enter card names')
+def click_on_add_a_card_button_enter_card_name(context, list_size, card_size, filename):
+    context.list_card = AddAListCardPage(context.driver)
+    list_val = int(list_size)
+    cards_val = int(card_size)
+    file_path = os.path.join(os.getcwd(), "resources")
+    context.filename = filename
+    context.cardfilepath = os.path.join(file_path, filename)
+
+    with open(context.cardfilepath, 'w') as file:
+        file.write('')
+
+    for i in range(list_val * cards_val):
+        card_name = fake.name()
+        with open(context.cardfilepath, 'a') as file:
+            file.write(f'{card_name}\n')
```

Comment:
If possible try to create a reusable method for writing/reading test data in File. You can just call the method and pass parameters like File path etc.

Reply:
Hi @ManojSavaram 
Here we are giving just a file path for reading or writing the data from the text file. So I think it is not  necessary to create a separate file and call in the step definition file